Makefile.toml

```diff
@@ -437,16 +446,6 @@ fi
 lz4 -df "${rootlz4}" "${root_image}"
 lz4 -df "${datalz4}" "${data_image}"
 
-# Canonicalize architecture identifier to the value recognized by EC2.
-case "${BUILDSYS_ARCH,,}" in
-   arm64|aarch64)
-      arch=arm64 ;;
-   x86_64|amd64)
-      arch=x86_64 ;;
-   *)
-      arch="${BUILDSYS_ARCH}" ;;
-esac
```

Comment:
If we don't already, we need to handle at least the `aarch64` -> `arm64` canonicalization in pubsys, since it's going away here.

Reply:
This is fixed by using the `parse_arch` function the SSM code already uses to automatically canonicalize the arch to the correct value.

Reply:
Fixed using the `parse_arch` function that automatically canonicalizes the architecture to the proper value.